sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1313,131 +1313,165 @@ object functions {
   //////////////////////////////////////////////////////////////////////////////////////////////
 
   /**
-   * Computes the cosine inverse of the given value; the returned angle is in the range
-   * 0.0 through pi.
```

Comment:
I am sorry if I missed a discussion about this. Why don't we keep this simple description for this method rather than removing out?

If we go this way, we should update both `functions.py` and `functions.R`. If you strongly prefer this, I would like to do this separately in another PR for both together.

Reply:
Seems like a valid point and I tend to agree with the conclusion to create a separate Jira for this. So far the discussion was about {{functions.scala}} but other languages should offer similar descriptions for the same thing. @srowen, can you please comment on this? I will take {{functions.scala}} out for now.